keyboards/zen/rev2/keymaps/default/keymap.c

```diff
@@ -0,0 +1,129 @@
+    KC_GESC,          KC_1,    KC_2,    KC_3,    KC_4,     KC_5,                       KC_6,     KC_7,     KC_8,    KC_9,    KC_0,    KC_BSPC, \
+    KC_TAB,           KC_Q,    KC_W,    KC_E,    KC_R,     KC_T,                       KC_Y,     KC_U,     KC_I,    KC_O,    KC_P,    KC_BSLS, \
+    LT(_NAV, KC_CAPS),KC_A,    KC_S,    KC_D,    KC_F,     KC_G,                       KC_H,     KC_J,     KC_K,    KC_L,    KC_SCLN, KC_QUOT, \
+    KC_LSFT,          KC_Z,    KC_X,    KC_C,    KC_V,     KC_B,    RGB_RMOD,RGB_MOD,  KC_N,     KC_M,     KC_COMM, KC_DOT,  KC_SLSH, KC_ENT,  \
+    KC_LCTL,          KC_LGUI, KC_LALT, RGB_TOG, MO(_NAV), KC_SPC,  KC_DEL,  KC_ENT,   KC_SPACE, MO(_NAV), KC_MINS, KC_EQL,  KC_PGUP, KC_PGDN, \
+                                                           KC_SPC,  KC_DEL,  KC_ENT,   KC_SPACE \
+  ),
+
+  /* NAV
+   * ,-----------------------------------------.              .-----------------------------------------.
+   * | GESC |   1  |   2  |   3  |   4  |   5  |              |   6  |   7  |   8  |   9  |   0  | Bksp |
+   * |------+------+------+------+------+------|              |------+------+------+------+------+------|
+   * | Tab  |   Q  |   W  |   E  |   R  |   T  |              |   Y  |   U  |   I  |   O  |   P  |   \  |
+   * |------+------+------+------+------+------|              |------+------+------+------+------+------|
+   * |CAPS(NAV)|A  |   S  |   D  |   F  |   G  |              |   H  |   J  |   K  |   L  |   ;  |  "   |
+   * |------+------+------+------+------+------+------..------+------+------+------+------+------+------|
+   * | Shift|   Z  |   X  |   C  |   V  |   B  |      ||      |   N  |   M  |   ,  |   .  |   /  |Enter |
+   * |------+------+------+------+------+------+------||------+------+------+------+------+------+------|
+   * | Ctrl | GUI  | Alt  |RGBTOG| NAV  |Space |Delete||Enter |Space |  NAV |   -  |   =  | PGUP | PGDN |
+   * `----------------------------------+-------------''------------------------------------------------'
+   *                                    |Space |Delete||Enter |Space |
+   *                                    '-------------''-------------'
+   */
+  [_NAV] = LAYOUT( \
+    KC_F1,   KC_F2,   KC_F3,   KC_F4,   KC_F5,   KC_F6,                     KC_F7,   KC_F8,    KC_F9,   KC_F10,  KC_F11,  KC_F12,  \
+    _______, RGB_SAI, RGB_VAI, RGB_SAD, RESET,   KC_LBRC,                   KC_RBRC, KC_PGUP,  KC_UP,   KC_PGDN, KC_INS,  KC_HOME, \
+    _______, RGB_HUD, RGB_VAD, RGB_HUI, _______, _______,                   _______, KC_LEFT,  KC_DOWN, KC_RGHT, KC_DEL,  KC_END,  \
+    KC_LSFT, _______, _______, _______, _______, _______, _______, _______, _______, _______,  _______, KC_MPLY, KC_MPRV, KC_MNXT, \
+    KC_LCTL, KC_LGUI, KC_LALT, RGB_MOD, _______, _______, _______, _______, _______, _______,  _______, KC_MUTE, KC_VOLU, KC_VOLD, \
+                                                 _______, _______, _______, _______
+  ),
+
+};
+
+#ifdef AUDIO_ENABLE
```

Comment:
Could this block be removed?  It's not needed,  as `set_single_persistent_default_layer` will actually handle this.

Reply:
Removed